guides/ManualLabellingSteps.md

```diff
@@ -104,17 +98,22 @@ If we are launching the tool for the first time for a new project:
   - Adjust their size by clicking and then dragging one of their corners
   - Delete the bounding boxes that don't apply anymore with `d`
 
-- You can also copy the bounding boxes to several previous or next frames (click the `Paste-n icon`` on the top bar and read the instructions) but I found this less useful
+- You can also copy the bounding boxes to several previous or next frames (click the `Paste-n icon` on the top bar and read the instructions)
+
+  - I found this less useful
+
 - Some convenient shortcuts:
 
   - Press `b` to toggle the boxes visibility
   - Press `l` to toggle the visibility of the boxes' IDs
   - Press `spacebar` to show the annotations for that frame
 
-- There is no undo action in the annotation tool!
+- **Important**: There is no undo action in the annotation tool!
```

Comment:
```suggestion
> [!CAUTION]
> There is no undo action in the annotation tool!
```

[Assuming this is only rendered in GFM](https://docs.github.com/en/get-started/writing-on-github/getting-started-with-writing-and-formatting-on-github/basic-writing-and-formatting-syntax#alerts).

Reply:
yes I was looking for this, thanks